lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -197,15 +212,20 @@ class CacheHeaders extends Audit {
         }
 
         const cacheControl = parseCacheControl(headers.get('cache-control'));
-        let cacheLifetimeInSeconds = CacheHeaders.computeCacheLifetimeInSeconds(
-          headers,
-          cacheControl
-        );
+        if (!this.shouldProcessRecord(headers, cacheControl)) {
+          continue;
+        }
 
-        // Ignore assets with an explicit no-cache policy
-        if (cacheLifetimeInSeconds === 0) continue;
+        // Ignore if cacheLifetimeInSeconds is defined and a nonpositive number.
```

Comment:
the `is defined` throws me for a loop when reading this. `Ignore if cacheLifetimeInSeconds is a nonpositive number` seems sufficient and is still a true description :)